### ISSUE TITLE
Use of skos:members

### ISSUE THREAD

**Issue by alanruttenberg** [ISO-TC211/GOM#14]
An old [pull request](https://github.com/ISO-TC211/GOM/pull/5) notes that skos:members is not a SKOS property and fixes it in one file. However a search of the repo finds 10 other mentions of it.

**Comment by nicholascar** [comment]
Hi @alanruttenberg. The PR you reference was one of mine - for a standard I cared about! I am now the new GOM convenor and I will be reviewing the generation of all ontologies.

Please see this test delivery of one set of terms - _roles_ - from one standard: https://github.com/ISO-TC211/GOM/tree/master/experimental. It is valid SKOS form and has been machine-validated against a formal profile of SKOS (the so-called [VocPub profile](https://w3id.org/profile/vocpub) for simple, single vocabulary publication). Any comments you have on this form of TC vocabs/code lists/taxonomic representations of standards' classes would be useful.

I will likely be using profiles of OWL, similar to VocPub but for ontologies, to validate ontologies soon too.

**Comment by alanruttenberg** [comment]
Hi @nicholascar,

Sorry for the delay in responding - I've been behind on work. I'll try to check this out this week. I'll also look through my notes to see if there are other things worth mentioning. I encountered the TC211 ontologies when I was doing a review of NEO, the NGA's ontology derived from NAS, which imports a number of them, although apparently doesn't really integrate them. NEO's on hold for the moment as there are discussions on NAS and ontology strategy.   I'm hoping that I'll be able to reboot development of NEO at some point but will have to wait on those deliberations. If you would like to chat informally some time about the TC211 ontologies generally I'm game for that. I'm interested in hearing about what's coming up and may have some time that I can devote to working through some of the issues.
